Project.toml

```
@@ -1,7 +1,7 @@
 name = "TrixiParticles"
 uuid = "66699cd8-9c01-4e9d-a059-b96c86d16b3a"
 authors = ["erik.faulhaber <44124897+efaulhaber@users.noreply.github.com>"]
-version = "0.1.3-dev"
+version = "0.1.4-dev"
```

```suggestion
version = "0.1.4-pre"
```
Do we want to use this? Just like in Trixi.jl? Why did we use `dev`?

---

Michael wrote I should use `-dev` when I pushed the 0.1.1 release.